### PR TITLE
Add slash to the ManifestDirPath

### PR DESCRIPTION
**Pull request by jebriede**
## Bug

Fixes: https://github.com/NuGet/Client.Engineering/issues/2124

Regression? Last working version: N/A

## Description
The insertion PRs are failing on SBOM checks because the path to the SBOM manifest file is missing a slash. This was previously working for some reason, but now the code that does the path discovery is failing to find the file and we need to fix the path.

## PR Checklist

- [x] PR has a meaningful title
- [x] PR has a linked issue.
- [x] Described changes

- **Tests**
  - [ ] Automated tests added
  - **OR**
  
  - [ ] Test exception
  - **OR**
  - [x] N/A 

- **Documentation**
  
  - [ ] Documentation PR or issue filled
  - **OR**
  - [x] N/A

### CODE REVIEW

**Review by zivkan — APPROVED** [review]

**Review by erdembayar — APPROVED** [review]

**Comment by jebriede** [comment]
This fix didn't seem to work at all. The initial PR build succeeded and then I ran an official build to check the SBOM results. The [14.GenerateVSManifestForVSIX.binlog](https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=7216272&view=artifacts&pathAsName=false&type=publishedArtifacts) shows the same warning as before and the path it's trying to evaluate didn't change, even though this change is adding a trailing slash to the end of the path.

**Review by donnie-msft — COMMENTED** [review]
It sounds like we're not sure this PR fixes the SBOM generation issue. I won't block the PR in case hotseat finds it does ultimately fix the issue.

Secondly, we should understand why other repos do not require the trailing slash. I found dotnet/razor has the same code we had (https://github.com/dotnet/razor/blob/main/eng/common/templates/steps/generate-sbom.yml#L11). I've asked current hotseat (@martinrrm) to look into this before we merge this PR.

**Comment by martinrrm** [comment]
Closing, this did not fix the SBOM failures.

**Pull request closed without merging**
